redaxo/src/core/lib/rex.php

```diff
@@ -181,7 +181,13 @@ public static function removeProperty($key)
      */
     public static function isSetup()
     {
-        return (bool) self::getProperty('setup', false);
+        static $setup;
```

Comment:
Ist das caching nötig?

Reply:
`rex::isSetup()` wird ja schon ab und zu aufgerufen (zum Beispiel für jede nicht gefundene Klasse).
Daher dachte ich, da es auch recht simpel ist, wäre es nicht verkehrt.
Wirklich wichtig wird es aber nicht sein. Kann es daher auch entfernen, wenn dir das lieber ist.

Reply:
Ich war hauptsächlich neugierig ob es bereits messbare fälle gab wo es nötig ist.

(ggf. Kann innerhalb eines requests das caching zu einem falschen ergebnis führen, wenn isSetup() aufgerufen wird, bevor der schalter aktiv umgedchaltet wird)

Reply:
Ich denke wirklich messbar wird es aktuell nicht sein. Und auch eher relevant, in irgendwelchen eher seltenen Konstellationen.

Ich würde es so lassen, solange es keine Probleme damit gibt.
Wichtig ist es mir aber nicht, kann es also auch gerne ändern.

Reply:
Wenn es nicht/noch nicht nötig ist würd ich es rauslassen